### PR TITLE
Adding UserAgent

### PR DESCRIPTION
**Pull request by smusali**

### CODE REVIEW

**Review by dchai76 — CHANGES_REQUESTED** [review]
Same suggestion as elsewhere, let's move this logic into `config`.

**Comment by smusali** [comment]
> Same suggestion as elsewhere, let's move this logic into `config`.

Done!

File: lib/logger.js [hunk 1]
```diff
@@ -140,7 +140,9 @@ function Logger(key, options) {
     this._req = {
         auth: { username: key }
         , agent: useHttps ? new Agent.HttpsAgent(configs.AGENT_SETTING) : new Agent(configs.AGENT_SETTING)
-        , headers: clone(configs.DEFAULT_REQUEST_HEADER)
+        , headers: Object.assign({}, clone(configs.DEFAULT_REQUEST_HEADER), {
```
Comment (dchai76) [inline lib/logger.js:143]: Why do you need the `{}` here? Won't `clone(configs.DEFAULT_REQUEST_HEADER)` give you something acceptable?

Comment (smusali) [reply lib/logger.js:143]: It's for making sure of nothing crazy is gonna happen in this merge of 2 objects - it's there just for safety reasons - no harm it has with its existence

**Review by dchai76 — APPROVED** [review]

**Pull request merged**